Rakefile

```diff
@@ -30,3 +30,24 @@ desc 'run project spec'
 task :project_spec do
   system('bundle exec parallel_rspec project_spec')
 end
+
+desc 'run tests in modified specs'
+task :in_modified_specs do
+  # get changes in framework
+  lib_diff = `git diff --name-only origin/master -- lib dockerfiles Dockerfile`
+  if lib_diff.empty?
+    # get changes in scripts and find them in spec
+    scripts_diff = `git diff --name-only origin/master -- js python | xargs -I {} grep -Rl {} spec`
```

Comment:
Maybe we can somehow do not use python here (maybe some bash magic)
So python will not required for this script

Reply:
This is not the use of Python, it's just a folder name where we store scripts for testing python wrapper

Reply:
Ah, got it, my bad :)